web/src/lib/components/shared-components/search-bar/search-filter-box.svelte

```diff
@@ -42,18 +42,22 @@
   const toStartOfDayDate = (dateString: string) => parseUtcDate(dateString)?.startOf('day').toISODate() || undefined;
   const dispatch = createEventDispatcher<{ search: SmartSearchDto | MetadataSearchDto }>();
 
+  function withoutNull<T>(value: T | null) {
```

Comment:
I'm not a huge fan of this. It adds a level of indirection just for a || or ?? operator. Just use the operator.

Reply:
How do you propose doing it instead? `country: searchQuery.country || undefined` doesn't work since both empty string, 0 and null map to `undefined`. `country: searchQuery.country ?? undefined` also doesn't work, since it maps `null` to `undefined`.